website/content/api-docs/system/rotate/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_label: Overview
```

Comment:
Oh, I see, `rotate/index.mdx` and `rotate.mdx` are the same path, so the redirect fails. I'd remove it for this path, pointing it at the index is probably OK.